report_templates/entitlements.erb

```diff
@@ -2,17 +2,27 @@
 name: Subscription - Entitlement Report
 snippet: false
 model: ReportTemplate
+template_inputs:
+- name: Days from Now
+  required: false
+  input_type: user
+  description: "Show subscriptions expiring in <= this many days.  Leave blank to show all entitlements."
+  advanced: false
+  value_type: plain
+  options: "7\r\n30\r\n60\r\n90\r\n120\r\n180"
```

Comment:
@ares Thanks, updated the template.  I'm not sure why I didn't come across this in my testing, maybe because I was manually pasting the ERB into the web UI instead of using `import_raw_template`?

I had to add a line
```
days_from_now = "" if days_from_now == 'no limit'
```
The Katello PR (just merged) was tuned to elegantly handle when `days_from_now` is an empty string, but broke everything when it was `'no limit'`.  Rather than introducing another Katello PR, I added the line here which seems to work fine.  Hopefully this is acceptable.

Reply:
> I was manually pasting the ERB into the web UI instead of using `import_raw_template`?

I guess you have defined the input manually and that allows empty string as input.

> I had to add a line

good thinking, I like that, let me retest this, if that works, I'm happy to merge

Reply:
@ares Did you get a chance to retest?